sample/FOSSASIA11/event

```diff
@@ -3,10 +3,10 @@
     "name": "FOSSASIA 2011",
     "latitude": 10.7627114,
     "longitude": 106.6911399,
-    "location_name": "Van Lang University Campus 2, 233A Phan Van Tri Street, Binh Thanh DistrictHo Chi Minh City",
-    "start_time": "2011-11-11T09:00:00",
-    "end_time": "2011-11-13T15:30:00",
-    "timezone": "UTC+07:00",
+    "location_name": "Van Lang University Campus 2, 233A Phan Van Tri Street, Binh Thanh District, Ho Chi Minh City",
+    "start_time": "2011-11-11T09:00:00+07:00",
+    "end_time": "2011-11-13T15:30:00+07:00",
+    "timezone": "Asia/Vietnum",
```

Comment:
@iamareebjamal I'm changing it to `Asia/Ho_Chi_Minh`
http://www.world-timedate.com/timezone/timezone_info_by_county.php?country_id=65